nixos/modules/virtualisation/containers.nix

```diff
@@ -29,12 +29,30 @@ in
     };
 
     cdi = {
-      dynamic.nvidia.enable = mkOption {
-        type = types.bool;
-        default = false;
-        description = lib.mdDoc ''
-          Enable dynamic CDI configuration for NVidia devices by running nvidia-container-toolkit on boot.
-        '';
+      dynamic.nvidia = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = lib.mdDoc ''
+            Enable dynamic CDI configuration for NVidia devices by running nvidia-container-toolkit on boot.
+          '';
+        };
+
+        mount-nvidia-binaries = mkOption {
```

Comment:
Libcuda is a binary too, maybe `executables` or `tools`

Reply:
+1, yeah, `tools` might be a more accurate name.